src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -189,7 +189,7 @@ AthensSurfaceExamples class >> example10 [
 		can setStrokePaint: Color red.
 		can draw ].
 		
-	Display getCanvas drawImage: 	surf asForm  at: 0@0
+	surf asForm asMorph openInWindow.
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?